lua/lspconfig/typescript.lua

```diff
@@ -0,0 +1,15 @@
+local M = {}
+
+function M.get_typescript_server_path(root_dir)
```

Comment:
for now, add this to util.lua.

we're trying to deprecate everything in that file, but it's helpful for this to live there until we figure out how to upstream things like this.